### PR TITLE
fix(components): Prevent stacked-bar errors when widget is not visible

### LINKED ISSUE #508: StackedBar: Errors thrown when height is 0
**What is happening?**
When the stacked bar widget has no width or height several erros are logged

**What should happen?**
We should control this case to prevent those errors.

### PR DESCRIPTION
**Pull request by IagoLast**
Fix #508 
---

### CODE REVIEW

File: packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx [hunk 1]
```diff
@@ -212,6 +212,9 @@ export class StackedBarWidget {
   }
 
   private _drawFigure() {
+    if (this._containerIsNotDrawable) {
```
Comment (rjimenezda) [inline packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx]: Uhhh...does this work without parenthesis?

```suggestion
    if (this._containerIsNotDrawable()) {
```

**Comment by rjimenezda** [comment]
:shipit:

File: packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx [hunk 2]
```diff
@@ -256,4 +259,12 @@ export class StackedBarWidget {
       this.el.forceUpdate();
     }
   }
+
+  // If container is not ready or the widget is invisible block drawing phase
+  private _containerIsNotDrawable() {
+    if (!this.container || this.el.clientWidth === 0 || this.el.clientHeight === 0) {
```
Comment (ivanmalagon) [inline packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx]: Do we have `this.el` always available when asking for their properties?

Comment (rjimenezda) [reply packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx]: We should, it's a decorated class property that gets assigned the DOM node of the element.

**Pull request merged**